.github/workflows/publish_sprite.yml

```
@@ -88,16 +88,21 @@ jobs:
           github_token: ${{ secrets.DEVTOOLS_GITHUB_TOKEN }}
           branch: ${{ github.ref }}
 
+      - name: Setup NPM Auth Token to .yarnrc.yml
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPMJS_PUBLISH_TOKEN }}
+        shell: bash
+        run: |
+          yarn config set npmRegistryServer "https://npm.pkg.github.com"
```

```suggestion
          yarn config set npmRegistryServer "https://npm.pkg.github.com"
```
В `.yarnrc.yml` уже указан верный регистри

---

Точно ⚡ https://github.com/VKCOM/icons/pull/588/commits/805ad859131093a0857ef24cbffaded5c11ae36a
Ещё добавил `tags: true` чтобы пушить код с тэгами.

И буду теперь rebase'ить